src/main/java/de/tum/in/www1/artemis/service/TutorLeaderboardService.java

```diff
@@ -99,6 +114,14 @@ else if (acceptedComplaintsView.getAcceptedComplaints() != null) {
                 }
             }
 
+            for (TutorLeaderboardAnsweredMoreFeedbackRequestsView answeredMoreFeedbackRequestsView : tutorLeaderboardAnsweredMoreFeedbackRequests) {
+                if (answeredMoreFeedbackRequestsView.getUserId().equals(tutor.getId())) {
+                    numberOfAnsweredMoreFeedbackRequests += answeredMoreFeedbackRequestsView.getAnsweredRequests();
+
+                    // TODO: decide how to count points for more feedback requests
```

Comment:
I guess this should not give any points, because it basically means that they only "repaired" the problem they introduced before.
We should on the other hand count "not resolved" More Feedback Requests with minus points (max points of the exercise). This motivates the tutors to resolve such requests as soon as possible